rct_optimizations/CMakeLists.txt

```diff
@@ -4,6 +4,12 @@ project(rct_optimizations VERSION 0.1.0 LANGUAGES CXX)
 find_package(rct_common REQUIRED)
 find_package(Ceres REQUIRED)
 
+find_package(catkin REQUIRED COMPONENTS
+  rct_common
+)
```

Comment:
`rct_optimizations` is a pure CMake package now, so make sure you don't add any ROS-specific stuff back in.